src/lingrex/trimming.py

```diff
@@ -115,9 +115,13 @@ def trimmed(self,
         e.g. morpheme boundaries.
         """
         if strategy in {'gap-oriented', 'gap'}:
-            candidates = [  # Sites with big enough gap ratio ordered by decreasing ratio.
-                idx for idx, score
-                in sorted(enumerate(self.gap_ratios), key=lambda x: x[1], reverse=True)
+            # Sites with big enough gap ratio, grouped by ratio, ordered by decreasing ratio.
+            candidates = [
+                [i[0] for i in idxs] for score, idxs in
```

Comment:
@LinguList  `candidates` is now a list of list of sites with the same gap ratio.